wallet/src/repository/wallet/mod.rs

```diff
@@ -496,6 +501,7 @@ where
         value: u64,
         fee: u64,
         recipient: Option<(types::PublicKeyHash, u64)>,
+        change_address_same_as_input: bool,
```

Comment:
Actually, when `recipient` is `None` this flag is set to `false`, and when `recipient` is `Some` this flag is `true`. But I didn't want to reuse this parameter because of unknown side effects.